beebop/app.py

```diff
@@ -111,12 +111,18 @@ def run_poppunk():
     """
     sketches = request.json['sketches'].items()
     project_hash = request.json['projectHash']
+    name_mapping = request.json['names']
```

Comment:
Looks like the schema needs updating here: https://github.com/bacpop/beebop_py/blob/main/spec/sketches.schema.json - if you add `additionalProperties: false` to the schema you'll get errors when you forget to keep them in sync